packages/web/src/components/common/pool-graph/PoolGraph.tsx

```diff
@@ -506,12 +506,12 @@ export const PoolGraphBinTooptip: React.FC<PoolGraphBinTooptipProps> = ({
           </span>
           <span className="amount total-amount">
             <MissingLogo symbol={tooltipInfo.tokenA.symbol} url={tooltipInfo.tokenA.logoURI} className="logo" width={20} mobileWidth={20} />
-            <span className="hidden">{tooltipInfo.tokenAAmount || "0"}</span>
+            <span className="hidden" style={(tooltipInfo.tokenAAmount || "0").length > 21 ? { fontSize: "12px"} : {}}>{tooltipInfo.tokenAAmount || "0"}</span>
```

Comment:
I think it would be better to separate the class name rather than writing it in the component style attribute, what do you think?

Reply:
DONE